Source/WebKit/Configurations/BaseTarget.xcconfig

```diff
@@ -50,6 +50,12 @@ ALTERNATE_HEADER_SEARCH_PATHS_iosmac = $(BUILT_PRODUCTS_DIR)$(WK_ALTERNATE_FRAME
 
 HEADER_SEARCH_PATHS = $(ALTERNATE_HEADER_SEARCH_PATHS) $(BUILT_PRODUCTS_DIR)/usr/local/include "$(WEBCORE_PRIVATE_HEADERS_DIR)/ForwardingHeaders" $(BUILT_PRODUCTS_DIR)/DerivedSources/WebKit $(WEBGPU_HEADER_SEARCH_PATHS) $(WEBKITADDITIONS_HEADER_SEARCH_PATHS) $(LIBWEBRTC_HEADER_SEARCH_PATHS) $(SRCROOT) $(HEADER_SEARCH_PATHS);
 
+PROFILE_DATA_PATH = $(PROFILE_DATA_PATH_PRODUCTION_$(DEPLOYMENT_LOCATION));
```

Comment:
Just to double check — `$(DEPLOYMENT_LOCATION)` here is set to `YES`/`NO` based on whether or not it's a production build?

Reply:
Hmm, this looks like it might be wrong for builds that install to device.

Reply:
I think Alexey is right: DEPLOYMENT_LOCATION should be YES for _any_ install build, including builds that install to device.

I wonder if the correct thing here would be to check the configuration, i.e. 

    DEPLOYMENT_LOCATION[config=Production] = $(SDK_DIR)...

Any objections to that?

Reply:
I see! Yeah, my followup question was going to be why we preferred that over using `$(CONFIGURATION)` or `config=Production`. It sounds like we'll want to switch to one of the two?

Reply:
No objections from me. I wonder if we have a way to test install to device builds in CI in the future.

Reply:
The new stack is:

    PROFILE_DATA_PATH = $(PROFILE_DATA_PATH_INTERNAL_$(USE_INTERNAL_SDK));
    PROFILE_DATA_PATH_INTERNAL_ = $(SRCROOT)/../../Tools/Profiling/Empty.profdata;
    PROFILE_DATA_PATH_INTERNAL_YES = $(BUILT_PRODUCTS_DIR)/usr/local/include/WebKitAdditions/Profiling/JavaScriptCore.profdata.compressed;
    PROFILE_DATA_PATH_INTERNAL_YES[config=Production] = $(SDK_DIR)/usr/local/include/WebKitAdditions/Profiling/JavaScriptCore.profdata.compressed;

Thanks for catching this!